gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/GradebookSpreadsheetFixedTables.html

```diff
@@ -1,28 +0,0 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
Now that categories are rendered by the `GbToolbarHeader` I can remove both `GradebookSpreadsheetFixedTables.java` and `GradebookSpreadsheetFixedTables.html`... they were kinda lame anyway.